rotkehlchen/globaldb/handler.py

```diff
@@ -1774,6 +1775,8 @@ def resolve_asset(identifier: str, form_with_incomplete_data: bool) -> AssetWith
         May raise:
         - UnknownAsset if the asset is not found in the database
         """
+        if identifier.startswith(NFT_DIRECTIVE):
```

Comment:
This and below are fixing something. Do you have a test for that? Can you please add for both?